resources/css/plugin.css

```diff
@@ -216,34 +216,48 @@
         display: inline-block;
     }
 
+    h1,h2,h3,h4,h5,h6 {
```

Comment:
Here and also below, please make sure there is a space after each comma `h1, h2`